components/ws-manager/pkg/manager/testdata/validateStartReq_invalidType.json

```diff
@@ -7,14 +7,14 @@
             "owner": "tester",
             "metaId": "foobar"
         },
-        "servicePrefix": "foobarservice",
+        "service_prefix": "foobarservice",
         "spec": {
             "ideImage": {
                 "webRef": "eu.gcr.io/gitpod-core-dev/buid/theia-ide:someversion"
             },
-            "workspaceImage": "eu.gcr.io/gitpod-dev/workspace-images/ac1c0755007966e4d6e090ea821729ac747d22ac/eu.gcr.io/gitpod-dev/workspace-base-images/github.com/typefox/gitpod:80a7d427a1fcd346d420603d80a31d57cf75a7af",
-            "checkoutLocation": "/",
-            "workspaceLocation": "/",
+            "workspace_image": "eu.gcr.io/gitpod-dev/workspace-images/ac1c0755007966e4d6e090ea821729ac747d22ac/eu.gcr.io/gitpod-dev/workspace-base-images/github.com/typefox/gitpod:80a7d427a1fcd346d420603d80a31d57cf75a7af",
```

Comment:
This looks fishy to me:
 - why the change to `workspace_image`?
 - and why do all other files keep using `workspaceImage`?
:thinking: